packages/editor-ui/src/components/Node.vue

```diff
@@ -256,6 +311,16 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 				this.$emit('duplicateNode', this.data.name);
 			});
 		},
+		getTriggerNodeTooltip (): string | undefined {
+			if (this.nodeType !== null && this.nodeType.hasOwnProperty('eventTriggerDescription')) {
+				return this.nodeType.eventTriggerDescription;
+			} else {
+				return `Waiting for you to create an event in ${this.getTrimmedTriggerNodeTypeName()}`;
+			}
+		},
+		getTrimmedTriggerNodeTypeName() {
+			return this.nodeType && this.nodeType.displayName.replace(/Trigger/, "");
+		},
```

Comment:
can you make these into one computed property.. it does not need to be a method...

Reply:
Done into one computed property.